src/cache.cpp

```diff
@@ -475,6 +475,10 @@ void Cache::update_lastmodified(const std::string& feedurl,
 			"empty, not updating anything");
 		return;
 	}
+
+	run_sql(prepare_query("INSERT OR IGNORE INTO rss_feed (rssurl, url, title) VALUES ('%q', '', '')",
+			feedurl));
```

Comment:
I prefer to use an upsert clause ([sqlite doc](https://www.sqlite.org/lang_upsert.html)), however, that is only available since sqlite 3.24.0 whereas Ubuntu 18.04 is stuck at sqlite 3.22.0.